src/workspaces/features/BucketExplorer/BucketExplorer.tsx

```diff
@@ -32,9 +37,36 @@ type BucketExplorerProps = {
 
 const BucketExplorer = (props: BucketExplorerProps) => {
   const { t } = useTranslation();
-  const { workspace, objects, perPage, onChangePage } = props;
+  const { workspace, objects, prefix, perPage, onChangePage } = props;
+  const clearCache = useCacheKey(["workspace", "files", prefix]);
+  const onDroppingFiles = (files: File[]) =>
+    uploader
+      .createUploadJob({
+        files: files,
+        async getXHROptions(file) {
```

Comment:
Tried for a while to leverage the onProgress callback to show a ProgressPie but it seems onProgress is not providing data and we have flickering when the file is small. Also we want to show the progress in a toast instead, so I don't want to invest much more time on this